File: shared/src/main/java/com/jawnnypoo/openmeh/shared/Video.java
package com.jawnnypoo.openmeh.shared;

import org.parceler.Parcel;

/**
 * The video of the day
 */
@Parcel
public class Video {
    String id;
    String startDate;
    String title;
    String url;
    Topic topic;

    public Video() {}

    public String getId() {
        return id;
    }

    public String getStartDate() {
        return startDate;
    }

    public String getTitle() {
        return title;
    }

    public String getUrl() {
        return url;
    }

    public Topic getTopic() {
        return topic;
    }
}

File: shared/src/main/java/com/jawnnypoo/openmeh/shared/model/Video.kt
package com.jawnnypoo.openmeh.shared.model

import com.squareup.moshi.Json

/**
 * The video of the day
 */
class Video {
    @Json(name = "id") var id: String? = null
    @Json(name = "title") var title: String? = null
    @Json(name = "url") var url: String? = null
}
